pkgs/development/cuda-modules/lib/default.nix

```diff
@@ -0,0 +1,11 @@
+{ lib }:
```

Comment:
Let's make it static?

```suggestion
let
  lib ? import ../../../../lib;
in
```

Reply:
Good suggestion -- implemented in latest.